src/case-ui-toolkit.module.ts

```diff
@@ -35,7 +35,7 @@ import { PipesModule, CaseReferencePipe, MarkdownComponent, MarkdownModule, Base
   WriteTextFieldComponent, WriteTextAreaFieldComponent, WritePhoneUKFieldComponent, WriteNumberFieldComponent, WriteEmailFieldComponent,
   WriteDateFieldComponent, ConditionalShowModule, ConditionalShowDirective, LabelSubstitutorModule,
   LabelSubstitutorDirective, CaseViewerComponent, CaseViewComponent, SearchFiltersModule, FocusElementModule,
-  FocusElementDirective, LoadingModule, PaginationModule, PaginationComponent, WriteCaseFlagFieldComponent } from './shared';
```

Comment:
@johnbenjamin-hmcts - just checking that the writable Case Flag field still works when this import is removed?

Reply:
@Daniel-Lam - WriteCaseFlagFieldComponent and ReadCaseFlagFieldComponent are used as entry components in PaletteModule and this module is imported in the CaseUIToolkitModule module.

Reply:
@Daniel-Lam - I can navigate to the screens when adding a new case flag from create case menu option and update case flag from the trigger dropdown.